www/index.html

```diff
@@ -15,12 +15,12 @@
             <h2 class="mtbs">Sign up</h2>
             <label
               class="form-label form-label-small"
-              for="username"
-            >Username</label>
+              for="Email"
+            >Email</label>
             <input
               class="form-input form-input-small mbm"
-              name="username"
-              id="username"
+              name="email"
+              id="email"
```

Comment:
I think we don’t need the `id="email"` here, it’s not used anywhere. Can you add `type="email"`? That will change the keyboard on touch devices.

same with the email input in the Log in form